src/content/docs/en/guides/migrate-to-astro/from-create-react-app.mdx

```diff
@@ -0,0 +1,379 @@
+
+## Add your CRA to Astro
+
+Your existing app can be rendered directly inside a new Astro project, often with no changes to your app's code.
+
+### Create a new Astro project
+
+Use the `create astro` command for your package manager to launch Astro's CLI wizard and select a new "empty" Astro project.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn create astro@latest
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+### Add integrations and dependencies
+Add the React integration using the `astro add` command for your package manager. If your app uses Tailwind or MDX, you can add multiple Astro integrations using the same command:
+
+```sh
+# Using NPM
+npx astro add react
+npx astro add react tailwind mdx
+# Using Yarn
+yarn astro add react
+yarn astro add react tailwind mdx
+# Using PNPM
+pnpm astro add react
+yarn astro add react tailwind mdx
+```
+
+If your CRA requires any dependencies (e.g. NPM packages), then install them individually using the command line or by adding them to your new Astro project's `package.json` manually and then running an install command. Note that many, but not all, React dependencies will work in Astro. 
+
+### Add your existing app files
+
+Copy your existing Create React App (CRA) project source files and folders (e.g. `components`, `hooks`, `styles`, etc.) into a new folder inside `src/`, keeping its file structure so your app will continue to work. Note that all `.js` file extensions must be renamed to `.jsx` or `.tsx`.
+
+Do not include any configuration files. You will use Astro's own `astro.config.mjs`, `package.json`, and `tsconfig.json`.
+
+Move the contents of your app's `public/` folder (e.g. static assets) into Astro's `public/` folder.
+
+<FileTree>
+- public/
+  - logo.png
+  - favicon.ico
+  - ...
+- src/
+  - cra-project/
+    - App.jsx
+    - ...
+  - pages/
+    - index.astro
+- astro.config.mjs
+- package.json
+- tsconfig.json
+</FileTree>
+
+### Render your app
+
+Import your app's root component in the frontmatter section of `index.astro`, then render the `<App />` component in your page template:
+
+```astro title="src/pages/index.astro"
+---
+import App from '../cra-project/App.jsx'
+---
+<App client:load/>
+```
+
+:::note[client directives]
+Your app needs a [client directive](/en/reference/directives-reference/#client-directives) for interactivity. Astro will render your React app as static HTML until you opt-in to client-side JavaScript.
+
+Use `client:load` to ensure your app loads immediately from the server, or `client:only="react"` to skip server-rendering and run your app entirely client-side.
```

Comment:
Nit: although server-side should be hyphened, that doesn't apply to server rendering as far as I know, so I would just change this to follow the SSR name pattern.

```suggestion
Use `client:load` to ensure your app loads immediately from the server, or `client:only="react"` to skip server-side rendering and run your app entirely client-side.
```